package.json

```diff
@@ -10,18 +10,20 @@
     "packages": [
       "packages/blade",
       "packages/token-publisher",
-      "packages/razorsharp"
+      "packages/razorsharp",
+      "packages/eslint-plugin-blade"
     ],
     "nohoist": [
       "**"
     ]
   },
   "scripts": {
-    "tsc:blade": "yarn workspace @razorpay/blade typecheck",
+    "tsc:blade": "lerna run --scope @razorpay/blade typecheck",
```

Comment:
why are we changing this to use lerna? 
Ideally we should stick to workspaces only, maybe later down the line we can remove lerna since lerna has a new successor (nx) now and isn't even being used very much in blade.

Reply:
For consistency https://github.com/razorpay/blade/pull/1495#discussion_r1281821818